src/Symfony/Component/Intl/Tests/DateFormatter/AbstractIntlDateFormatterTest.php

```diff
@@ -398,8 +398,8 @@ public function testFormatWithDateTimeZone()
             $this->markTestSkipped('Only in PHP 5.5+ IntlDateFormatter allows to use DateTimeZone objects.');
         }
 
-        if (defined('HHVM_VERSION_ID')) {
-            $this->markTestSkipped('This test cannot work on HHVM. See https://github.com/facebook/hhvm/issues/5875 for the issue.');
+        if (defined('HHVM_VERSION_ID') || PHP_VERSION_ID <= 50509) {
```

Comment:
the PHP_VERSION_ID check is actually 3 lines above. Just update it with the right version (I thought it was 5.5.0+)